apps/deploy-web/src/components/new-deployment/ManifestEdit.tsx

```diff
@@ -47,15 +48,18 @@ type Props = {
   selectedTemplate: TemplateCreation | null;
   editedManifest: string | null;
   setEditedManifest: Dispatch<SetStateAction<string>>;
+  github?: boolean;
+  setGithub?: Dispatch<boolean>;
 };
 
-export const ManifestEdit: React.FunctionComponent<Props> = ({ editedManifest, setEditedManifest, onTemplateSelected, selectedTemplate }) => {
+export const ManifestEdit: React.FunctionComponent<Props> = ({ editedManifest, setEditedManifest, onTemplateSelected, selectedTemplate, github }) => {
   const [parsingError, setParsingError] = useState<string | null>(null);
   const [deploymentName, setDeploymentName] = useState("");
   const [isCreatingDeployment, setIsCreatingDeployment] = useState(false);
   const [isDepositingDeployment, setIsDepositingDeployment] = useState(false);
   const [isCheckingPrerequisites, setIsCheckingPrerequisites] = useState(false);
   const [selectedSdlEditMode, setSelectedSdlEditMode] = useAtom(sdlStore.selectedSdlEditMode);
+  const [isRepoDataValidated, setIsRepoDataValidated] = useState(false);
```

Comment:
`validated` sounds like data was validated leaving validity question unanswered. `data` may mean many things, I suggest `input` as it's a bit more specific. You're welcome to find even better name if you like.

```suggestion
  const [isRepoInputValid, setIsRepoInputValid] = useState(false);
```